tests/unittests/test_child_bookmarks.py

```diff
@@ -23,13 +27,14 @@ def test_child_bookmarking(self, mock_now, mock_get_catalog_entry, mock_event_up
         mock_get_catalog_entry.return_value = {'tap_stream_id': 'invoices', 'schema': {}, 'key_properties': [], 'metadata': [{"valid-replication-keys": ["created"]}]}
         # metadata.to_map return value
         mock_to_map.return_value = {(): {'table-key-properties': ['id'], 'selected': True, 'forced-replication-method': 'INCREMENTAL', 'valid-replication-keys': ['created']}}
-        invoice_line_items_ts = 1641137533
-        Context.state = {"bookmarks": {"invoices": {"date": 1645716195}, "invoice_line_items": {"date": invoice_line_items_ts}}}
+        invoice_line_items_ts = 1641137533 # 02-01-2022T03:32:13Z
+        Context.state = {"bookmarks": {"invoices": {"date": 1641137533}, "invoice_line_items": {"date": invoice_line_items_ts}}}
         sync()
         stop_window = dt_to_epoch(now_time)
```

Comment:
Even if we keep the stop window 30 days, still the loop will execute one time, and as we just want to check that the bookmark is used properly we can keep any stop window, it will not affect anywhere in this testcase

Reply:
I understand. However, it would be good if we check the routine scenarios in the test case rather than mocking it unneccesarily